.circleci/config.yml

```diff
@@ -182,7 +193,7 @@ workflows:
   build:
     jobs:
       - build
-  updateEapBuildNumber:
```

Comment:
I'd use `updateEapBuildNumber` actually &mdash; it clearly indicates that something is well, **updated** in the repository (that a side effect happens)